Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -206,6 +206,7 @@ view push state =
           driverMapsHeaderView push state
         , rideActionModelView push state
         ]
+      , if state.props.rcDeactivePopup then PopUpModal.view (push <<< RCDeactivatedAC) (driverRCPopUpConfig state) else dummyTextView
```

Comment:
not related to your PR but can you also add `NAMMAYATRI` merchant check for `autoPayBannerView`, as it should should be visible only for `NY` and not `JS`

Reply:
done but not tested bro

Reply:
@ashishsingh101 instead of keeping merchant checks let's start using configurations for the same. In code it will be hard to maintain check based on merchants in future

Reply:
@S-PRAVINKUMAR without testing try not to raise the PR in any case

Reply:
sure bro.